pom.xml

```diff
@@ -1602,7 +1616,7 @@ under the License.
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-shade-plugin</artifactId>
-					<version>3.0.0</version>
+					<version>3.1.1</version>
```

Comment:
please move this into the jdk9 profile; I don't have the time right now to verify that the shading still works correctly for all modules.